src/eventsub/event/websocket.rs

```diff
@@ -62,6 +62,9 @@ pub struct SessionData<'a> {
     /// The UTC date and time that the connection was created.
     #[serde(borrow = "'a")]
     pub connected_at: Cow<'a, types::TimestampRef>,
+    /// Undocumented field
+    #[serde(borrow = "'a")]
+    pub recovery_url: Option<Cow<'a, str>>,
```

Comment:
Could you mention what this is for? Especially in comparison to `reconnect_url`. There's also the `unsupported` feature - not sure if this should be used here.

Reply:
I'd love to but I have no idea what it's for. I'll do some digging

Reply:
@Nerixyz twitch dev on discord said he doesn't know what it does and can be ignored. Idk what proper resolution is here so let me know what you think. What is the `unsupported` feature you speak of?

Reply:
Barry is not a twitch developer, he develops stuff using twitch ;)

Maybe lets just put this field `#[doc(hidden)]`.

> There's also the unsupported feature - not sure if this should be used here.

No that would defeat the purpose of the pr to fix deny unknown fields.


```suggestion
    #[doc(hidden)]
    #[serde(borrow = "'a")]
    pub recovery_url: Option<Cow<'a, str>>,
```